public/app/features/plugins/admin/pages/Browse.tsx

```diff
@@ -56,45 +58,47 @@ export default function Browse(): ReactElement | null {
     <Page navModel={navModel}>
       <Page.Contents>
         <PluginPage>
-          <HorizontalGroup justify="space-between">
+          <HorizontalGroup wrap>
             <SearchField value={q} onSearch={onSearch} />
-            <HorizontalGroup>
-              <RadioButtonGroup
-                value={filterByType}
-                onChange={onFilterByTypeChange}
-                options={[
-                  { value: 'all', label: 'All' },
-                  { value: 'datasource', label: 'Data sources' },
-                  { value: 'panel', label: 'Panels' },
-                  { value: 'app', label: 'Applications' },
-                ]}
-              />
-              <RadioButtonGroup
-                value={filterBy}
-                onChange={onFilterByChange}
-                options={[
-                  { value: 'all', label: 'All' },
-                  { value: 'installed', label: 'Installed' },
-                ]}
-              />
-              <Select
-                width={24}
-                value={sortBy}
-                onChange={onSortByChange}
-                options={[
-                  { value: 'name', label: 'Sort by name (A-Z)' },
-                  { value: 'updated', label: 'Sort by updated date' },
-                  { value: 'published', label: 'Sort by published date' },
-                  { value: 'downloads', label: 'Sort by downloads' },
-                ]}
-              />
+            <HorizontalGroup wrap className={styles.actionBar}>
+              <div>
+                <RadioButtonGroup
```

Comment:
For small screens this component is not being rendered properly, I think we need to add the `flex:wrap` as well, right now is hardcoded as `no-wrap`, maybe we can pass down the property?

![minorIssuePluginsCatalog](https://user-images.githubusercontent.com/239999/127621174-aae04685-3698-433f-9177-46969278dd30.png)

Reply:
Thanks for pointing this out @axelavargas. I'm not sure I should introduce changes to the `@grafana/ui` RadioButtonGroup component in this PR as it's used in ~37 places throughout the app and I'm worried I might introduce bugs elsewhere and bloat this PR further.